cfgov/cfgov/settings/base.py

```diff
@@ -469,12 +469,15 @@
     'fast.fonts.net',
     'tagmanager.google.com',
     'optimize.google.com',
-    'api.mapbox.com')
+    'api.mapbox.com',
+    'fonts.googleapis.com')
```

Comment:
Trivial: Include trailing comma to avoid modifying this line next time we add a new one :)

Bonus points: alphabetize the list by TLD, then subdomain.

Reply:
I agree that someone ought to alphabetize these some day ;)